WcaOnRails/db/migrate/20161212200704_add_maldives_to_countries.rb

```diff
@@ -0,0 +1,9 @@
+class AddMaldivesToCountries < ActiveRecord::Migration
+  def up
+    execute "INSERT INTO `Countries` (`id`, `name`, `continentId`, `iso2`) VALUES ('Maldives', 'Maldives', '_Asia', 'MV')"
```

Comment:
That is true, but I'd rather keep this as a copy paste of the line from `WcaOnRails/db/seeds/countries.seeds.rb`.